entries/g/gleam.io.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Gleam": {
+    "domain": "gleam.io",
+    "tfa": [
+      "totp",
+      "custom-software"
+    ],
+    "custom-software": [
+      "Authy"
```

Comment:
Do you know if Gleam uses "Authy Powered Tokens" or just authenicator tokens? If it's the latter then we can drop the custom software from the listing as they were probably just suggesting it as a 2FA app in the documentation.

Reply:
After testing it, I found it to be the latter.

![image](https://github.com/user-attachments/assets/ed79caf4-7ad8-4128-b957-c246d95efbda)

Reply:
Nice work! I assume you have reset your 2FA after sending that screenshot (if not I would). But just a tip for the future, I would blur out any sensitive data like 2FA setup QR codes, email addresses, etc. when providing evidence in a PR or issue on here.